### PR TITLE
Problem: Events are insufficient for monitoring gravity transfer

### PR DESCRIPTION
**Pull request by calvinaco**
Solution: Emit event after handling SendToCosmosEvent at x/gravity module

### CODE REVIEW

**Review by leejw51crypto — APPROVED** [review]

File: module/x/gravity/keeper/ethereum_event_handler.go [hunk 1]
```diff
@@ -44,6 +45,19 @@ func (k Keeper) Handle(ctx sdk.Context, eve types.EthereumEvent) (err error) {
 		if err := k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, addr, coins); err != nil {
 			return err
 		}
+
+		ctx.EventManager().EmitEvent(sdk.NewEvent(
```
Comment (yihuang) [inline module/x/gravity/keeper/ethereum_event_handler.go:49]: Is it possible to implement in the hook, so we don't need to patch gravity-bridge?

Comment (calvinaco) [reply module/x/gravity/keeper/ethereum_event_handler.go:49]: Yes I think so. But not sure if we want to patch the other places with weird unit64 to int cast as well.

Alternatively I can implement the event emit on Cronos hook first; then create a PR to the upstream and we can patch them in next iteration. What do you think?

**Review by yihuang — APPROVED** [review]

File: module/x/gravity/keeper/ethereum_event_handler.go (hunk 1 above)
Comment (calvinaco) [reply module/x/gravity/keeper/ethereum_event_handler.go:49]: Open this PR on Cronos side: https://github.com/crypto-org-chain/cronos/pull/133

See which way we want to go

**Review by tomtau — COMMENTED** [review]
should this be merged -- or is https://github.com/crypto-org-chain/cronos/pull/133 enough?

**Comment by thomas-nguy** [comment]
Is it ported to peggyJV repo?

**Pull request closed without merging**